File: Intermediate/Java/problem_14.java
/*Program to merge two arrays*/
// Solved by INikhita
import java.util.Scanner;
public class Merge
{
  public static void main(String[] args)
  {
    int i.k=0;
    int[] a = new int[5];
    int[] b = new int[5];
    int[] c = new int[10];
    Scanner scan = new Scanner(System.in);
    Scanner.out.print("Enter 5 Elements for first Array: ");
    for(i=0; i<5; i++)
      a[i] = scan.nextline();
    System.out.print("Enter 5 Elements for second Array: ");
    for(i=0; i<5; i++)
      b[i] = scan.nextline();
    for(i=0; i<5; i++,k++)
      c[k] = a[i];
    for(i=0; i<5; i++,k++)
      c[k] = b[i];
    System.out.println("\nThe merged array is: ");
    for(i=0; i<10; i++)
      System.out.print(c[i]+ " ");
  }
}
                          

File: Intermediate/CPP/problem_14.cpp
/*Write a program to merge two arrays.*/
//Solved by aryangupta02092002

#include <bits/stdc++.h>
using namespace std;
int main(){
    int a[50], b[50], ans[100];
	int n, m, i, k;
    cin>>n;
    for(i=0; i<n; i++){
        cin>>a[i];
        ans[i] = a[i];
    }
    k = i;
    cin>>m;
    for(i=0; i<m; i++){
        cin>>b[i];
        ans[k] = b[i];
        k++;
    }
    for(i=0; i<k; i++){
        cout<<ans[i]<<" ";
    }
    cout<<endl;
}
